.github/workflows/build.yml

```diff
@@ -46,8 +46,7 @@ jobs:
             ruby: '3.0'
             command: bin/rake smoke_test
           - name: windows
-            # These two fail because of broken stacktraces on windows: minitest_node_nodejs mspec_nodejs
-            command: bundle exec rake rspec minitest_nodejs
+            command: bundle exec rake
```

Comment:
I would suggest adding another pipeline perhaps, because I think this particular one may run for too long

Reply:
done